doc/agentservice.md

```diff
@@ -0,0 +1,11 @@
+# Agent service
+Agent service is responsible for serving collected data to the user. Once user create a source for his vCenter environment the Agent service provide a streaming service to download OVA image that is ready to be booted on the vCenter enviroment to run the collection of the data.
+
+## Agent API
+There are two APIs related to the Agent.
+
+### Internal API
```

Comment:
can you also update the api dir to include 2 sub directory (now there is only one and the other openapi is stored in the api dir )

Reply:
Well, I think it's kind good designed. It's like root API + agent api, so there is root folder with openapi spec and subfolder with agent api.

Reply:
I just think that it might confusing what is agent related to
agent service agent VM